### PR TITLE
docs: fix `&hellip;` presented in aio code docs

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
the symbol `&hellip;` displays correctly in code-example blocks and in
plain html text, it does however not work correctly for code represented
using `backticks`, fix such occurrences by replacing `&hellip;` with `...`

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

Sometimes the `&hellip;` is displayed in the ui, as for example:

aot-before

forms-before

## What is the new behavior?

`...` is being displayed instead:

aot-after

froms-after

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

@josmar-crwdstffng, please have a look :slightly_smiling_face: 

As you can see in the description above, the issue is when we have the `&hellip;` inside backticks, I fixed it by simply replacing them with `...`. Alternatively I could replace the backticks with code-example blocks, but I am afraid that would get more verbose... please let me know what you prefer

### CODE REVIEW

File: aio/content/guide/i18n-common-translation-files.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ The following actions describe the translation process for French.
 
     The `id="introductionHeader"` is a [custom ID][AioGuideI18nOptionalManageMarkedText], but without the `@@` prefix required in the source HTML.
 
-1.  Duplicate the `<source>&hellip; </source>` element in the text node, rename it to `target`, and then replace the content with the French text.
+1.  Duplicate the `<source>...</source>` element in the text node, rename it to `target`, and then replace the content with the French text.
```
Comment (josmar-crwdstffng) [inline aio/content/guide/i18n-common-translation-files.md:143]:
```suggestion
1.  Duplicate the `<source>... </source>` element in the text node, rename it to `target`, and then replace the content with the French text.
```

File: aio/content/guide/npm-packages.md [hunk 2]
```diff
@@ -123,9 +123,9 @@ The following `devDependencies` are provided in the default `package.json` file
...
 | `@angular/compiler-cli`                                                   | The Angular compiler, which is invoked by the Angular CLI's `ng build` and `ng serve` commands. |
-| `@types/&hellip; `                                                        | TypeScript definition files for 3rd party libraries such as Jasmine and Node.js.                |
-| `jasmine/&hellip; `                                                       | Packages to support the [Jasmine](https://jasmine.github.io) test library.                      |
-| `karma/&hellip; `                                                         | Packages to support the [karma](https://www.npmjs.com/package/karma) test runner.               |
+| `@types/...`                                                        | TypeScript definition files for 3rd party libraries such as Jasmine and Node.js.                |
+| `jasmine/...`                                                       | Packages to support the [Jasmine](https://jasmine.github.io) test library.                      |
```
Comment (josmar-crwdstffng) [inline aio/content/guide/npm-packages.md:127]:
```suggestion
| `jasmine/... `                                                       | Packages to support the [Jasmine](https://jasmine.github.io) test library.                      |
```

**Review by josmar-crwdstffng — COMMENTED** [review]
Thank you for your contribution.
Overall the replacements match the intent.
I placed a few suggestions for you to implement before I approve this PR.

**Review by josmar-crwdstffng — APPROVED** [review]

**Review by josmar-crwdstffng — APPROVED** [review]

**Comment by josmar-crwdstffng** [comment]
@global-docs-approvers LGTM

**Review by gkalpak — APPROVED** [review]
Thx, @dario-piotrowicz!

Reviewed-for: global-docs-approvers

<sub>(Adding another global approval, because PullApprove needs a specific format in order to recognize the approval :smiley:)</sub>

**Pull request closed without merging**